pkg/cloud/services/provider/provider.go

```diff
@@ -84,7 +84,9 @@ func newClient(cloud clientconfig.Cloud, caCert []byte) (*gophercloud.ProviderCl
 	if cloud.Verify != nil {
 		config.InsecureSkipVerify = !*cloud.Verify
 	}
-	config.RootCAs.AppendCertsFromPEM(caCert)
+	if caCert != nil {
+		config.RootCAs.AppendCertsFromPEM(caCert)
+	}
```

Comment:
@sbueringer What would happen if in this case caCert is nil?

Reply:
The cacert is not added? :)

So either you set a cacert then it's also used to verify the OpenStack Certs or you don't and it isn't used (only the rootCAs)

Reply:
@sbueringer Got it thanks :+1:

Reply:
that's my bad, actually I had a mandatory cacert requirement but agree it's optiponal